projects/cdk/schematics/ng-add/steps/add-taiga-icons.ts

```diff
@@ -19,7 +20,7 @@ export function addTaigaIcons(options: Schema): Rule {
 
 export function addTaigaAssetsToAngularJson(options: Schema): Rule {
     return updateWorkspace(workspace => {
-        const project = getProject(options, workspace);
+        const project = getProject(options, workspace as unknown as WorkspaceDefinition);
```

Comment:
Did you update (locally) `@schematics/angular/utility/workspace` after angular update ?

Probably, we dont need typecasting here (if `@schematics/angular` has the same version with `@angular-devkit`).

Reply:
fixed